### PR TITLE
add wistia video and copy to end of airflow branch operator guide

### PR DESCRIPTION
**Pull request by medinab**

### CODE REVIEW

File: guides/airflow-branch-operator.md [hunk 1]
```diff
@@ -124,3 +124,17 @@ class ShortCircuitOperator(PythonOperator, SkipMixin):
...
+<iframe src="https://fast.wistia.net/embed/iframe/9c4267f3e4" title="branchpythonoperator Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" allowfullscreen msallowfullscreen width="100%" height="450"></iframe>
+
+There is much more to the BranchPythonOperator than simply choosing tasks over others. 
+What if you want to trigger your tasks only on specific days? and not on holidays? 
+What if you want to trigger a DAG Run only if the previous one has succeeded?
+Discover how to solve those incredible common use cases by getting access to the following course for free:
+
+[https://academy.astronomer.io/branching-course](https://academy.astronomer.io/branching-course)
+
+See you there! ❤️
```
Comment (paolaperaza) [inline guides/airflow-branch-operator.md:140]:
```suggestion
## Beyond the BranchPythonOperator

<!-- markdownlint-disable MD033 -->
<iframe src="https://fast.wistia.net/embed/iframe/9c4267f3e4" title="branchpythonoperator Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" allowfullscreen msallowfullscreen width="100%" height="450"></iframe>

There is much more to the BranchPythonOperator than simply choosing one task over another.

- What if you want to trigger your tasks only on specific days? And not on holidays? 
- What if you want to trigger a DAG Run only if the previous one succeeded?

 For more guidance and best practices on common use cases like the ones above, try out Astronomer's [Academy Course on Branching](https://academy.astronomer.io/branching-course) for free today.

See you there! ❤️
```

**Review by paolaperaza — COMMENTED** [review]
@medinab Added a copy suggestion here, let me know what you think & I'll come back to approve + merge!

File: guides/airflow-branch-operator.md (hunk 1 above)
Comment (medinab) [reply guides/airflow-branch-operator.md:140]: updated!

**Review by paolaperaza — APPROVED** [review]

**Comment by paolaperaza** [comment]
Sweet, thanks @medinab ! Merging.

**Pull request merged**